### DISCUSSION TITLE
Why does Sionna use image method？

### DISCUSSION THREAD

**Discussion by fighting-xq** [NVlabs/sionna#664] [General]
Image method brings exponentially-increasing computation complexity. However, SBR seems to be a more reasonable approach.

**Comment by faycalaa** [comment]
Hi @qiangdadi,

To compute channel impulse responses, Sionna actually uses both the image method and SBR (shooting and bouncing rays). It first uses SBR to find candidate paths, and then evaluates and corrects these candidates using the image method. This approach avoids evaluating all possible candidates using the image method, and therefore prevents exponential scaling with the number of primitives.

**Reply by fighting-xq** [reply]
I also want to ask why Sionna use both image method and SBR. Does SBR can not find paths correctly? And why? Thanks for your reply and Forgive my naive question.

**Reply by jhoydis** [reply]
Hi @qiangdadi,

SBR is used as a heuristic to find potential sequences of primitives in the scene. The image method is then used to figure out if the is a chain of exact specular paths that connect a transmitter and a receiver.

SBR alone cannot find exact specular paths because the probability that a randomly shot ray hits a point is zero.

**Reply by fighting-xq** [reply]
Thanks a lot!
